File: 1 - Iniciante/C99/BEE1048.c
#include <stdio.h>

int main(void){
	float salario;
	int reajuste;
	
	scanf("%f", &salario);
	
	if (salario >= 0 && salario <= 400)
		reajuste = 15;
	else if (salario > 400 && salario <=800)
		reajuste = 12;
	else if (salario > 800 && salario <= 1200)
		reajuste = 10;
	else if (salario > 1200 && salario <= 2000)
		reajuste = 7;
	else 
		reajuste = 4;
		
	printf("Novo salario: %.2f\n", salario + (salario * ((float) reajuste/100)));
	printf("Reajuste ganho: %.2f\n", salario * ((float) reajuste/100));
	printf("Em percentual: %d %%\n", reajuste);
		
	return 0;
}

File: 1 - Iniciante/Python 3.11/BEE1048.py
salary = float(input())

if (salary<=400):
    adjust = 0.15
elif (salary>400 and salary<=800):
    adjust = 0.12
elif (salary>800 and salary<=1200):
    adjust = 0.10
elif (salary>1200 and salary<=2000):
    adjust = 0.07
else:
    adjust = 0.04

new = salary + salary * adjust

print(f'Novo salario: {new:.2f}')
print(f'Reajuste ganho: {new - salary:.2f}')
print(f'Em percentual: {adjust * 100:.0f} %')